core/primitives/address.cpp

```diff
@@ -4,7 +4,170 @@
  */
 
 #include "primitives/address.hpp"
+#include "codec/leb128/leb128.hpp"
+#include "common/visitor.hpp"
+#include "cppcodec/base32_default_rfc4648.hpp"
+#include "crypto/blake2/blake2b.h"
 
-namespace fc::primitives {
-  // implementation
+#include <memory>
```

Comment:
wrong place. 
Proper order of includes:
```
#include "primitives/address.hpp"

#include <memory>

#include <cppcodec/base32_default_rfc4648.hpp>
#include "codec/leb128/leb128.hpp"
#include "common/visitor.hpp"
#include "crypto/blake2/blake2b.h"
```

Reply:
Sure. Will check what google has to say about this aspect of code style